HHS/Head-Start-TTADP
# [TTAHUB-2572] Add groups to TR participants

## Description of change

This change add the ability to select groups on the **TR > session > participants** page. This follows the same logic as the AR group selection. This serves to simply auto populate the recipients. If the group recipents are altered a information message is shown asking if they want to start over.

## How to test

- Review all code
- Create a group for a select region add some recipients.
- Find and Event for the same region and select the group
Verify: All recipients populate and you can save the report
- Do the same thing as the above two steps but modify (add/remove) recipients.
Verify: You should get an information message saying group recipients have been modified and you can reset.

- You should still be able to edit an existing TR session participants without issue.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2572


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
